CMakeLists.txt

```diff
@@ -943,6 +943,13 @@ if (NOT BOOTSTRAPPING_MODE)
   message(FATAL_ERROR "turning off bootstrapping is not supported anymore")
 endif()
 
+endif()
+
+# Disable bootstrapping when we aren't building SwiftSyntax
```

Comment:
This comment is not very helpful - it just describes what one can read from the next two source lines, anyway.

Can you add more context here? I assume that `SWIFT_BUILD_SWIFT_SYNTAX` is the main "switch" which tells if host tools are available.
If so, why not call it `SWIFT_HOST_TOOLS_AVAILABLE` or something like that. It feels like `SWIFT_BUILD_SWIFT_SYNTAX` is abused for that purpose.